src/SFA.DAS.Tools.Support.Web/App_Start/AuthorizationService.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.Tools.Support.Web.Infrastructure;
+
+namespace SFA.DAS.ToolService.Web.AppStart
+{
+    public static class AuthorizationService
+    {
+        public static void AddAuthorizationService(this IServiceCollection services)
+        {
+            const string serviceClaimType = "http://service/service";
+
+            services.AddAuthorization(options =>
+                {
+                    options.AddPolicy(PolicyNames.HasTier2Tier3Account, policy =>
+                    {
+                        policy.RequireAuthenticatedUser();
+                        policy.RequireClaim(serviceClaimType, new string[] { UserClaims.SCP, UserClaims.SCS});
+                    });
+                    options.AddPolicy(PolicyNames.HasTier3Account, policy =>
+                    {
+                        policy.RequireAuthenticatedUser();
+                        policy.RequireClaim(serviceClaimType, UserClaims.SCP);
+                    });
+                    options.AddPolicy(PolicyNames.HasTier2Account, policy =>
+                    {
+                        policy.RequireAuthenticatedUser();
+                        policy.RequireClaim(serviceClaimType, UserClaims.SCS);
+                    });
```

Comment:
this policy is unnecessary and can be covered by the overarching rule already in place on startup to ensure that the required service claims exist to allow the user access to any page of the site